### PR TITLE
[+] Technical - On start, create a forestadmin.json file that contains the apimap definition

### PR DESCRIPTION
**Pull request by VincentMolinie**
## Pull Request checklist:

- [x] Write an explicit title for the Pull Request
- [x] Write changes made in the CHANGELOG.md
- [ ] Create automatic tests
- [ ] No automatic tests failures
- [ ] Test manually the implemented changes
- [ ] Review my own code (indentation, syntax, style, simplicity, readability)
- [ ] Wonder if you can improve the existing code

### CODE REVIEW

**Comment by VincentMolinie** [comment]
[(5) Liana - On start, Rails liana should create a forestadmin.json file that contains the apimap definition.](https://trello.com/c/zFiz0JiI/3101-5-liana-on-start-rails-liana-should-create-a-forestadminjson-file-that-contains-the-apimap-definition)

**Comment by ArnaudValensi** [comment]
Let's wait arnaudb on monday to answer our questions.

File: lib/forest_liana/bootstraper.rb [hunk 1]
```diff
@@ -174,6 +178,12 @@ def create_apimap
       end
     end
 
+    def create_apimap_json
+      File.open(File.join(Rails.root, 'forestadmin-schema.json'), 'w') do |f|
+        f.puts JSON.pretty_generate(ForestLiana.apimap.as_json)
+      end
```
Comment (ArnaudValensi) [inline lib/forest_liana/bootstraper.rb]: We should print a clear message if:

    the file is invalid
    the file is empty
    the file is not present

Comment (VincentMolinie) [reply lib/forest_liana/bootstraper.rb]: Not in this PR we are just creating the file

File: lib/forest_liana/bootstraper.rb [hunk 2]
```diff
@@ -38,6 +38,10 @@ def generate_apimap
       create_apimap
       require_lib_forest_liana
       format_and_validate_smart_actions
+
+      unless Rails.env.production?
```
Comment (arnaudbesnier) [inline lib/forest_liana/bootstraper.rb]: `if Rails.env.development?` is a better condition.
What if the server is in `staging` environment or any other unknown type.

File: lib/forest_liana/bootstraper.rb [hunk 3]
```diff
@@ -38,6 +38,10 @@ def generate_apimap
...
+        create_apimap_json
```
Comment (arnaudbesnier) [inline lib/forest_liana/bootstraper.rb]: `create_apimap_json` is more `update_generated_schema`

**Pull request merged**